app/controllers/activitypub/inboxes_controller.rb

```diff
@@ -52,6 +53,20 @@ def upgrade_account
     DeliveryFailureTracker.reset!(signed_request_account.inbox_url)
   end
 
+  def process_collection_synchronization
+    raw_params = request.headers['X-AS-Collection-Synchronization']
+    return if raw_params.blank?
+    return if ENV['DISABLE_FOLLOWERS_SYNCHRONIZATION'] == 'true'
```

Comment:
I'd unite this with the line above. Maybe extract to `disable_followers_synchronization?` method for easier testing.

Reply:
Put it on one line, not sure about a different method.